application/src/main/java/be/jpendel/application/PersonMapper.java

```diff
@@ -6,7 +6,9 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class PersonMapper {
+/* TODO : mark: PersonMapper en PersonDTO moeten volgens mij in de domain layer
+*/
+class PersonMapper {
```

Comment:
stel nu dat je de use cases met bijhorende services in een apparte package steekt per applicatie. Misschien wel aparte jars van maakt. Dan wil je toch niet je ze allemaal moet redeployen omdat Person verandert ?
Is dit geen open closed issue ?